src/python-common/ceph/rgw/rgwam_core.py

```diff
@@ -246,11 +253,26 @@ def create(self, realm: EntityKey, zg: EntityKey = None, endpoints=None, is_mast
 
         return json.loads(stdout)
 
+    def modify(self, realm: EntityKey, zg: EntityKey, endpoints=None):
+        ze = ZoneEnv(self.env, realm=realm, zg=zg)
+        params = ['zonegroup', 'modify']
+        opt_arg(params, '--endpoints', endpoints)
+        return RGWAdminJSONCmd(ze).run(params)
```

Comment:
```
ceph/rgw/rgwam_core.py:262:1: E302 expected 2 blank lines, found 1
```
this stuff can be found locally by running tox in src/python-common btw

Reply:
Fixed, thanks for the tip :+1: